crates/core/src/lib.rs

```diff
@@ -1,9 +1,9 @@
 pub mod kafka;
 pub mod logger;
-pub mod types;
-
 #[cfg(any(test, feature = "test-utils"))]
 pub mod test_utils;
+pub mod types;
+pub mod user_ops_types;
```

Comment:
nit: could we put these under `pub mod logger`?

Reply:
Sure, curious what is the reasoning ?